jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -116,7 +110,7 @@
   	private Map<String, Boolean> packageAssertionStatus;
   	private Map<String, Boolean> classAssertionStatus;
   	/*[IF Sidecar19-SE]*/
-  	private final Hashtable<String, NamedPackage> packages = new Hashtable();
+    private final Hashtable<String, NamedPackage> packages = new Hashtable<>();
```

Comment:
tab vs space? Formatting seems off here

Reply:
The lines before and after have (some) spaces and I assumed it was four of them so I made this line match. Upon closer inspection, the lines above and below start with two spaces and a tab. I suggest any further cleanup should be done separately.

Reply:
Agreed.